### PR TITLE
feat(perf-issues): Add link to transaction summary from Span Evidence

### PR DESCRIPTION
**Pull request by mjq-sentry**
In the `Transaction` line of Span Evidence, link the transaction title value to the transaction summary page for that transaction.

Screenshot 2023-03-07 at 4 46 23 PM

The link takes you to the Transaction Summary page, e.g. https://try.sentry-demo.com/organizations/major-panther/performance/summary/?project=183242&query=&referrer=performance-transaction-summary&statsPeriod=7d&transaction=checkout&unselectedSeries=p100%28%29

Screenshot 2023-03-07 at 4 48 45 PM

### CODE REVIEW

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 1]
```diff
@@ -238,7 +257,21 @@ const PresortedKeyValueList = ({data}: {data: KeyValueListData}) => (
...
 
-const makeTransactionNameRow = (event: Event) => makeRow(t('Transaction'), event.title);
+const makeTransactionNameRow = (event: Event, orgSlug: string) => {
+  const to = {
+    pathname: normalizeUrl(`/organizations/${orgSlug}/performance/summary/`),
+    query: {
+      project: event.projectID,
+      transaction: event.title,
+    },
+  };
```
Comment (DominikB2014) [inline static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:267]: We have a way to do this actually
https://github.com/getsentry/sentry/blob/29b76b87c07c6e70f44a52dd2bfd7cf3a6fff928/static/app/views/performance/table.tsx#L206
I'd probably opt to do it that way, and for query you can just make it blank `query: {query: ''}`
which should leave it to default

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 2]
```diff
@@ -36,12 +39,14 @@ type SpanEvidenceKeyValueListProps = {
...
   event: EventTransaction;
   offendingSpans: Span[];
+  orgSlug: string;
   parentSpan: Span | null;
 };
 
 const TEST_ID_NAMESPACE = 'span-evidence-key-value-list';
 
 export function SpanEvidenceKeyValueList({event}: {event: EventTransaction}) {
+  const {slug: orgSlug} = useOrganization();
```
Comment (DominikB2014) [inline static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:49]: Just wanted to note this,
Tbh, idk what the best practice for this is, in a lot of component we pass the organization prop around from the parent.

**Review by gggritso — APPROVED** [review]
⛵

**Review by DominikB2014 — COMMENTED** [review]
Couple comments left, lgtm overall

**Review by udameli — APPROVED** [review]

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx (hunk 1 above)
Comment (mjq-sentry) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx]: Awesome, thanks! Originally went looking for something like this and couldn't find it. Replaced to use it :+1:

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx (hunk 2 above)
Comment (mjq-sentry) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:49]: We discussed this offline - going to leave as is for now.

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx [hunk 3]
```diff
@@ -51,6 +52,12 @@ describe('SpanEvidenceKeyValueList', () => {
       expect(
         screen.getByTestId('span-evidence-key-value-list.transaction')
       ).toHaveTextContent('/');
+      expect(
+        screen.getByTestId('span-evidence-key-value-list.transaction').querySelector('a')
+      ).toHaveAttribute(
+        'href',
+        `/organizations/org-slug/performance/summary/?project=123&referrer=performance-transaction-summary&transaction=%2F&unselectedSeries=p100%28%29`
```
Comment (DominikB2014) [inline static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx:59]: @mjq-sentry fyi this referrer is not correct, I didn't realize that `transactionSummaryRouteWithQuery` would add it.
I think we should add a referrer prop, to `transactionSummaryRouteWithQuery`. I can make that PR quickly.

Comment (mjq-sentry) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx:59]: It's okay, I went through the exact same thought process 😅 Came to the same conclusion in the end too - I think it's (somewhat surprisingly) correct as is.

**Pull request merged**